core/Symbols.cc

```diff
@@ -968,6 +976,12 @@ ClassOrModule::findMemberFuzzyMatchConstant(const GlobalState &gs, NameRef name,
     absl::c_reverse(result);
     return result;
 }
+std::pair<int, int> ClassOrModule::caseInsensitiveAndSensitiveDist(std::string_view s1, std::string_view s2,
+                                                                   int bound) noexcept {
+    int sensitive = Levenstein::distance(s1, s2, bound);
+    int insensitive = Levenstein::distance(absl::AsciiStrToLower(s1), absl::AsciiStrToLower(s2), bound);
```

Comment:
hm, this is already one of the slowest parts of sorbet, and we're now calling it twice... is there a way we can not do that?

Reply:
ack

let me try and make a modified Levenstein distance which simultaneously does both